src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/RetrieveOrganisationByOrgIdTest.java

```diff
@@ -76,7 +76,7 @@ public void rdcc117_ac2_pui_organisation_manager_can_retrieve_organisation_by_or
 
     @Test
     public void rdcc117_ac3_user_without_appropriate_permission_cannot_retrieve_organisation_by_orgIdentifier_for_external() {
-        Map<String, Object> response = professionalApiClient.retrieveOrganisationByOrgIdExternal(HttpStatus.FORBIDDEN, generateBearerTokenForUser("caseworker-caa"));
+        Map<String, Object> response = professionalApiClient.retrieveOrganisationByOrgIdExternal(HttpStatus.FORBIDDEN, generateBearerTokenForUser(hmctsAdmin));
```

Comment:
here as well no need merge the changes from master ..demo one is the correct one

Reply:
This method is already there , and added as a part of AAC functionality